example/src/requirements.txt

```diff
@@ -0,0 +1,4 @@
+PyYAML
+pika
+pycti
+IPy
```

Comment:
This docker image build is not deterministic/reproducible. Adding the version of dependencies could be a good idea  :smiley: